telegram/ext/commandhandler.py

```diff
@@ -49,7 +49,7 @@ class CommandHandler(Handler):
         :class:`telegram.ext.CommandHandler` does *not* handle (edited) channel posts.
 
     Attributes:
-        command (:obj:`str` | List[:obj:`str`]): The command or list of commands this handler
+        command (SLT[:obj:`str`]): The command or list of commands this handler
```

Comment:
Could you change those to ``:obj:`str` | List/Tuple[:obj:`str`]`` or ``:class:`telegram.utils.types.SLT`[:obj:`str`]``? People don't know what `SLT` is and will get confused